Project.toml

```diff
@@ -3,14 +3,18 @@ uuid = "5049e819-d29b-5fba-b941-0eee7e64c1c6"
 version = "0.3.0"
 
 [deps]
+ADNLPModels = "54578032-b7ea-4c30-94aa-7cbd1cce6c9a"
```

Comment:
Let's use Requires.

Reply:
> > 145 problems from the PureJuMP submodule are missing here, and 3 problems here are not in PureJuMP.
> 
> Do you think you could make a list of those problems in an issue?

No problem, I will do it as soon as this one is merged (unless you prefer to list to be completed first?)